share/src/main/java/io/zensoft/share/dto/VacancyDto.java

```diff
@@ -26,4 +26,5 @@
     private int salaryMin;
     private int salaryMax;
     private String image;
+    private String facebookUserAccessToken;
```

Comment:
I think token shouldnt be transfered with Vacancy object

Reply:
Good point. However, this solution has both pros and cons. If access token and vacancy are transferred in different objects, there will be problems with synchronization, because access token should be renewed every hour. So we have to pass access token with every post request to Facebook.